compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeRootRegistry.skiko.kt

```diff
@@ -78,13 +78,4 @@ internal class ComposeRootRegistry : PlatformContext.RootForTestListener {
     fun getComposeRoots(): Set<PlatformRootForTest> {
         return synchronized(lock) { roots.toSet() }
     }
-
-    fun <R> withRegistry(block: () -> R): R {
```

Comment:
Some exaplanation here:

calling `withRegistry` and passing a block with`kotlinx.coroutines.test.runTest` call is not correct for web - because `kotlinx.coroutines.test.runTest` returns immediately.

So I split the logic from this function:
- setupRegistry is called before the scene is created and before calling `kotlinx.coroutines.test.runTest`
- tearDownRegistry is called inside of `kotlinx.coroutines.test.runTest` block (in finally)